README.md

```diff
@@ -1 +1,87 @@
 # VTEX Pages editor
+
+The Pages editor it's platform to dynamically edit a VTEX Store, making possible to select editable components and change its configurations adding or removing content in a simple and straightforward away.
+
+## How to make your Component editable
+
+The editor supports two ways of defining an editable component, thought a static schema structure or a dynamic function, that receives data and create the schema to be displayed.
+
+### Static Schema
+
+Add to your component an `schema` constant, a JSON object that with the following structure:
+
+```
+const schema = {
+  type: 'object',
+  title: 'The component title',
+  properties: {
+    property1: {
+      type: 'string'
+      title: 'Title of the property'
+    },
+    ...{n* properties}
+  }
+}
+```
+
+The property type can be: `String`, `Object` or `Number`, if `Object` it will have the same structure as the parent properties.
+
+### Dynamic Schema
+
+Add to your component a function `getSchema`, that will have the logic to dynamically create the schema need to build your component structure. The Page Editor will call that function each time that the page form has a change to its state, that enables to add and remove fields from the schema, like the example bellow.
```

Comment:
⛏  to create the schema dynamically need to build your component structure.
**which enables **

Reply:
👍